lib/fog/openstack/models/volume/volume.rb

```diff
@@ -6,8 +6,10 @@ class OpenStack
       class Volume < Fog::OpenStack::Model
         identity :id
 
-        attribute :display_name,        :aliases => 'displayName'
-        attribute :display_description, :aliases => 'displayDescription'
+        # NOTE: The attributes "name" and "description" are called
+        # "display_name" and "display_description" in API version v1.
+        attribute :name,                :aliases => ['display_name', 'displayName', :display_name]
+        attribute :description,         :aliases => ['display_description', 'displayDescription', :display_description]
```

Comment:
might be cleaner to create v1 and v2 models, with extracting the same behavior to base class